src/wui/map_tags.cc

```diff
@@ -24,33 +24,73 @@
 
 namespace {
 
-const std::map<std::string, std::string> kMapTags = {
-   /** TRANSLATORS: This is a map tag */
-   {"official", _("Official")},
-   /** TRANSLATORS: This is a map tag */
-   {"unofficial", _("Unofficial")},
-   /** TRANSLATORS: This is a map tag */
-   {"balanced", _("Balanced")},
-   /** TRANSLATORS: This is a map tag */
-   {"unbalanced", _("Unbalanced")},
-   /** TRANSLATORS: This is a map tag */
-   {"seafaring", _("Seafaring")},
-   /** TRANSLATORS: This is a map tag */
-   {"ferries", _("Ferries")},
-   /** TRANSLATORS: This is a map tag */
-   {"artifacts", _("Artifacts")},
-   /** TRANSLATORS: This is a map tag */
-   {"scenario", _("Scenario")},
-   /** TRANSLATORS: This is a map tag */
-   {"ffa", _("Free for all")},
-   /** TRANSLATORS: This is a map tag. One versus one. */
-   {"1v1", _("1v1")},
-   /** TRANSLATORS: This is a map tag */
-   {"2teams", _("Teams of 2")},
-   /** TRANSLATORS: This is a map tag */
-   {"3teams", _("Teams of 3")},
-   /** TRANSLATORS: This is a map tag */
-   {"4teams", _("Teams of 4")},
+const std::map<std::string, TagTexts> kMapTags = {
+   {"official",
+    {/** TRANSLATORS: This is a map tag */
+     gettext_noop("Official"),
```

Comment:
Yes, we already have a few instances of it. I also tried `utils/buildcat.py` on this because I was unsure at one point about whether some translator comments would turn out right, and it worked nicely.

Reply:
ok